cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -1,183 +1,189 @@
-{% import 'molecules/notification.html' as notification %}
+{% import 'molecules/notification.html' as notification with context %}
 
 {% macro render(value, settings, form_id) %}
 
     {% set form = settings.form %}
 
     {# JS-disabled result handling #}
-    {% for message in get_messages(request) %}
-        {{ notification.render(message.tags.split(' ')[0], true, message.message) }}
-    {% endfor %}
+    
+<div class="o-feedback">
+	<form method="post"
+	      action="."
+	      class="o-form
+	             block
+	             {% if value.was_it_helpful_text %}
+	             block__bg
+	             block__border
+	             block__padded-top
+	            {% endif %}">
+	<div class="u-mb15">
+		{% if get_messages(request) %}
+			{% for message in get_messages(request) %}
+		        {{ notification.render(message.tags.split(' ')[0], true, message.message) }}
+		    {% endfor %}
+	    {% else %}
+	    	{{ notification.render('success', false, '') }}
+	    {% endif %}
+    </div>
+	<div class="form-l">
+	    <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
+	    <input type="hidden" name="form_id" value="{{ form_id }}">
+	    <input type="hidden" name="referrer" value="{{ request.META.HTTP_REFERER or '' }}">
 
-<form method="post"
-      action="."
-{% if not value.was_it_helpful_text %}
-      class="o-form">
-{% else %}
-      class="o-form
-             block
-             block__bg
-             block__border
-             block__padded-top">
-{% endif %}
-<div class="form-l">
-    <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
-    <input type="hidden" name="form_id" value="{{ form_id }}">
-    <input type="hidden" name="referrer" value="{{ request.META.HTTP_REFERER or '' }}">
+	    {% if value.was_it_helpful_text %}
+	    <fieldset class="o-form-input-group o-form-input-group__two-column">
+	        <div class="form-l_col
+	                    form-l_col-1
+	                    form-l_col__flush-bottom">
+	            <legend class="h4">
+	                {{ value.was_it_helpful_text }}
+	            </legend>
+	        </div>
+	        <div class="m-big-radio-button">
+	            <div class="m-big-radio-button_container">
+	                <input class="cf-input"
```

Comment:
@sebworks Good point -- I do think we should be using the input macros. Right now there are differences between their layout and the design for this form, though. It seems like this is something we could revisit and either standardize the form designs or make the input macros a bit more flexible to accommodate some differences in design.